src/games/strategy/triplea/Constants.java

```diff
@@ -144,6 +144,7 @@
   String UNIT_TYPE_CARRIER = "carrier";
   String UNIT_TYPE_FIGHTER = "fighter";
   String UNIT_TYPE_BOMBER = "bomber";
+  String UNIT_TYPE_TAC_BOMBER = "tactical_bomber";
```

Comment:
It's more information hiding. The rest of the system should only know about or see what it absolutely must, that details such as constant values etc are at the lowest layer possible.

Of some note, if this value is a constant, it's still hardcoded IMO, just not extracted to a constant in a Constants class. IE: you still need a code change and recompile and redeploy, hence hardcoded.